python/examples/basic/Getting_Started.ipynb

```diff
@@ -75,7 +75,7 @@
    "outputs": [],
    "source": [
     "# Note: you may need to restart the kernel to use updated packages.\n",
-    "%pip install whylogs"
+    "%pip install whylogs==1.3.29.dev0"
```

Comment:
undo edits to the Getting Started example if not needed, we don't want to pin the version to a dev build

Reply:
Will do in a follow-up PR immediately after this is merged